app/controllers/runs_controller.rb

```diff
@@ -56,11 +56,7 @@ def update
 
     if params[@run.id36][:srdc_url]
       srdc_id = SpeedrunDotCom::Run.id_from_url(params[@run.id36][:srdc_url])
-      if !srdc_id
-        redirect_params = {
-          alert: 'Your Speedrun.com URL must have the format https://www.speedrun.com/tfoc/run/6yjoqgzd.'
-        }
```

Comment:
Now when I enter `6yjoqgzd` into this field, my Speedrun.com link gets set to `https://www.speedrun.com/run/f`.

Reply:
Looks like it was saving the string representation of false.  Should be fixed now good catch.